csm/common/conf.py

```diff
@@ -79,16 +79,21 @@ def decrypt_conf():
         if not cluster_id:
             raise ClusterIdFetchError("failed to get cluster id.")
         for each_key in const.DECRYPTION_KEYS:
-            cipher_key = Cipher.generate_key(cluster_id,
-                                             const.DECRYPTION_KEYS[each_key])
+            # This Change is Specifically for SMS1 -> SMS2 where CSM Password
+            # was Hardcoded.
             encrypted_value = Conf.get(const.CSM_GLOBAL_INDEX, each_key)
-            try:
-                decrypted_value = Cipher.decrypt(cipher_key,
-                                                 encrypted_value.encode("utf-8"))
-                Conf.set(const.CSM_GLOBAL_INDEX, each_key,
-                        decrypted_value.decode("utf-8"))
-            except CipherInvalidToken as error:
-                raise CipherInvalidToken(f"Decryption for {each_key} Failed. {error}")
+            if not encrypted_value and each_key == "CSM.password":
+                Log.info("Setting default password for csm user.")
+                decrypted_value = const.NON_ROOT_USER.encode("utf-8")
```

Comment:
Is it encode or decode ?

Reply:
Hi @ujjwalpl ,

Here we are fetching the hardcoded CSM password hence here it is `encode` since the value recieved from `py-utils decrypt `  is in utf-8 format and is decoded later below.